src/app/shared/components/competition-card/competition-card.component.spec.ts

```
@@ -4,12 +4,20 @@
 import { StsConfigLoader } from 'angular-auth-oidc-client';
 import { NgxsModule, Store } from '@ngxs/store';
 import { TranslateModule } from '@ngx-translate/core';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'HttpClientTestingModule'. Consider to delete it or rename it.

'HttpClientTestingModule' is defined here:


(lint/suspicious/noRedeclare)